public/style.css

```diff
@@ -1,200 +1,200 @@
-	width: 9rem;
+  width: 9rem;
 }
 .del-address {
-	width: 7rem;
+  width: 7rem;
 }
 #add-address:hover, #add-phone-number:hover,
 .del-phone-number:hover, .del-address:hover {
-    background: linear-gradient(to bottom, #adadad 5%, #c0c0c0 100%);
-    background-color: #8f8f8f;
+  background: linear-gradient(to bottom, #adadad 5%, #c0c0c0 100%);
+  background-color: #8f8f8f;
 }
 #sub:hover {
-    background: linear-gradient(to bottom, #2D77A2 5%, #337DA8 100%);
-    background-color: #28739E;
+  background: linear-gradient(to bottom, #2D77A2 5%, #337DA8 100%);
+  background-color: #28739E;
 }
 .list-contact {
-	margin-left: 1%;
-	clear: both;
-	height: 20rem;
-	overflow: auto;
-	margin-bottom: 2%;
+  margin-left: 1%;
+  clear: both;
+  height: 20rem;
+  overflow: auto;
+  margin-bottom: 2%;
 }
 .list-contact .add-bar {
-	margin-left: 0.05%;
-	font-size: 1.5rem;
-	margin-bottom: 1.5%;
+  margin-left: 0.05%;
+  font-size: 1.5rem;
+  margin-bottom: 1.5%;
 }
 .list-contact ul {
-	margin-left: 1rem;
+  margin-left: 1rem;
 }
 .list-contact li {
-	padding: 0.25rem;
-	width: 18rem;
-	display: flex;
-	flex-direction: row;
+  padding: 0.25rem;
+  width: 18rem;
+  display: flex;
+  flex-direction: row;
 }
 .list-contact li:hover {
-	cursor: pointer;
-	box-shadow: 2px 2px 10px #808080;
+  cursor: pointer;
+  box-shadow: 2px 2px 10px #808080;
 }
 .list-contact li:hover svg {
-	display: inline-block;
+  display: inline-block;
 }
 .list-contact a {
-	flex-grow: 3;
-	text-decoration: none;
-	color: #216288;
-	width: 17rem;
+  flex-grow: 3;
+  text-decoration: none;
+  color: #216288;
+  width: 17rem;
 }
 .list-contact h4 {
-	display: inline-block;
+  display: inline-block;
 }
 .list-contact .plus {
-	vertical-align: middle;
+  vertical-align: middle;
 }
 .list-contact .plus:hover {
-	cursor: pointer;
+  cursor: pointer;
 }
 .list-contact .x {
-	display: none;
-	flex-grow: 1;
-	vertical-align: middle;
+  display: none;
+  flex-grow: 1;
+  vertical-align: middle;
 }
 .visually-hidden {
-    position: absolute;
-    left: -10000px;
-    top: auto;
-    width: 1px;
-    height: 1px;
-    overflow: hidden;
+  position: absolute;
+  left: -10000px;
+  top: auto;
+  width: 1px;
+  height: 1px;
+  overflow: hidden;
 }
 .edit-item-form input {
-	width: 100%;
-    background-color: #F8F8F8;
-    padding: 2px;
+  width: 100%;
+  background-color: #F8F8F8;
+  padding: 2px;
 }
 h4 {
-	font-size: 1.2rem;
-	font-weight: 300;
-    color: #C7CACD;
-    font-style: italic;
+  font-size: 1.2rem;
+  font-weight: 300;
+  color: #C7CACD;
+  font-style: italic;
 }
 .show-contact {
-	font-weight: 600;
+  font-weight: 600;
 }
 .show-contact > ul {
-	list-style-type: none;
+  list-style-type: none;
 }
 .show-contact > ul > li{
-	margin: 1% 0;
+  margin: 1% 0;
 }
 .show-contact > ul > ul {
-	list-style-type: disc;
+  list-style-type: disc;
 }
 .show-contact > ul > ul > li {
-	margin: 2% 0 2% 10%;
+  margin: 2% 0 2% 10%;
 }
 .hidden {
-	display: none;
+  display: none;
 }
```

Comment:
You may want to add comments at least to your functions to help you out if you ever come back to this code.